backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/process/C_BPartner_Stats_ComputeBPartnerStats.java

```diff
@@ -21,7 +24,8 @@ public class C_BPartner_Stats_ComputeBPartnerStats extends JavaProcess implement
 	protected String doIt()
 	{
 		final I_C_BPartner_Stats stats = getRecord(I_C_BPartner_Stats.class);
-		final BPartnerStats bpStats = Services.get(IBPartnerStatsDAO.class).getCreateBPartnerStats(stats.getC_BPartner());
+		final I_C_BPartner statsPartner = InterfaceWrapperHelper.load(BPartnerId.ofRepoId(stats.getC_BPartner_ID()), I_C_BPartner.class);
```

Comment:
Don't always write InterfaceWrapperHelper. 
Instead, please, do some static imports for the exact methods you need, for example "import static org.adempiere.model.InterfaceWrapperHelper.load;"